runtime/kusama/src/lib.rs

```diff
@@ -1163,7 +1164,13 @@ impl parachains_paras_inherent::Config for Runtime {
 	type WeightInfo = weights::runtime_parachains_paras_inherent::WeightInfo<Runtime>;
 }
 
-impl parachains_scheduler::Config for Runtime {}
+impl parachains_scheduler::Config for Runtime {
+	type CoreAssigners<T: runtime_parachains::scheduler::Config> = (
+		runtime_parachains::scheduler_parachains::ParachainsScheduler,
```

Comment:
Given that parathreads aren't used in Kusama scheduling, I suggest that this should just be the `ParachainsScheduler`. We can create a separate PR to activate the Parathreads scheduler in the Kusama and Polkadot runtimes, which is more in line with what we usually do: small kusama/polkadot runtime PRs to activate features after the bulk of the code has already been introduced.